src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -1007,8 +1010,17 @@ var ReactDOMFiberComponent = {
               nextProp,
             );
           } else {
-            // $FlowFixMe - Should be inferred as not undefined.
-            extraAttributeNames.delete(propKey.toLowerCase());
+            let ownNamespace = parentNamespace;
+            if (ownNamespace === HTML_NAMESPACE) {
```

Comment:
Why rename this `ownNamespace` here instead of using `parentNamespace` directly or setting `ownNamespace` as the argument?

Reply:
Disregard. Sorry I didn't catch the code below.